x-pack/docs/en/rest-api/security/has-privileges.asciidoc

```diff
@@ -30,6 +30,14 @@ privilege is assigned to the user.
 `privileges`::: (list) A list of the privileges that you want to check for the
 specified indices.
 
+`application`::
+`application`::: (string) The name of the application.
+`privileges`::: (list) A list of the privileges that you want to check for the
+specified resources. May be either application privilege names, or the names of
+actions that are granted by those privileges
+`resources`::: (list) A list of resource names against which the privileges
+should be checked
```

Comment:
This description and example aren't enough to explain application privileges, in my opinion. We should add a link to a page that explains the concept in more detail.